### ISSUE TITLE
Clarify/simplify Windows Pause image

### ISSUE THREAD

**Issue by TBBle** [kubernetes/website#29056]
**This is a Feature Request**

**What would you like to be added**

![image](https://user-images.githubusercontent.com/138397/126480140-0748eac6-f56b-437e-802f-30247be98906.png)

This is not totally clear, in terms of answering the question "What should my pause image be?".

**Why is this needed**

Listing `mcr.microsoft.com/oss/kubernetes/pause:3.4.1` first suggests it's primary, but:
1. It is listed as 'Windows' but also supports the linux-amd64 platform, according to its Manifest List (see comments)
1. `mcr.microsoft.com/oss/kubernetes/pause:3.5` exists ([reference](https://mcr.microsoft.com/v2/oss/kubernetes/pause/tags/list)), but isn't mentioned here, suggesting latency in updating the docs.
2. `k8s.gcr.io/pause:3.5` covers more architectures, so can support a superset of the hosts that `mcr.microsoft.com/oss/kubernetes/pause:3.5` supports.
3. The relationship between the two isn't clear. Are they the same source? Or just the same version by lucky happenstance? Why would I chose one over the other?

**Comments**

(Forked out of #26335, which was just about getting the version bumped)

It might be worth working out if the MCR-hosted pause image is diverged from the GCR-hosted pause image in any particular way, or was just built for fewer targets, and either way, the page could be updated to refer to `mcr.microsoft.com/oss/kubernetes/pause:3.5`.

I'd also probably list MCR second, since I assume the intention is that GCR-hosted image becomes the primary one, unless a specific use-case for the MCR-hosted image is known that means it should be preferred in some circumstances.

<details>

<summary> The Manifest Lists for reference </summary>

`curl -H "Accept: application/vnd.docker.distribution.manifest.list.v2+json" https://k8s.gcr.io/v2/pause/manifests/3.5`

```json
{
   "schemaVersion": 2,
   "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
   "manifests": [
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 526,
         "digest": "sha256:369201a612f7b2b585a8e6ca99f77a36bcdbd032463d815388a96800b63ef2c8",
         "platform": {
            "architecture": "amd64",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 526,
         "digest": "sha256:94ee124c4b0ca7c1315c06c31532f78a929051ae8da7f122f905b2cbbfb1ecba",
         "platform": {
            "architecture": "arm",
            "os": "linux",
            "variant": "v7"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 526,
         "digest": "sha256:76ca2030ac3433ab5bbcfdea286b0876b129fc276e0f9d2811674141ea7bab6b",
         "platform": {
            "architecture": "arm64",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 526,
         "digest": "sha256:95e406e45b39993dc89f964ab1ab42db30369bd3406c1b11ef2129ab18d9c7bb",
         "platform": {
            "architecture": "ppc64le",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 526,
         "digest": "sha256:fc5aae77765f7f26f729bad7bfb3cef1bf4e9260d616d173af4506b4f9fc63d3",
         "platform": {
            "architecture": "s390x",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 1158,
         "digest": "sha256:f3c77e5959c2e376825fc9b7cb5e591f5837792d1bf18de0119a46b6a98ea9d9",
         "platform": {
            "architecture": "amd64",
            "os": "windows",
            "os.version": "10.0.17763.1817"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 1158,
         "digest": "sha256:40ba27d6c61be49daebcd869d86d4f9f2b499b4d1feb606ee530dd4f86dedf6c",
         "platform": {
            "architecture": "amd64",
            "os": "windows",
            "os.version": "10.0.18362.1256"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 1158,
         "digest": "sha256:8c0853e6c27bb685dd2ff57616c122dd7265e150502ea8083010281621d404cb",
         "platform": {
            "architecture": "amd64",
            "os": "windows",
            "os.version": "10.0.18363.1440"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 1158,
         "digest": "sha256:125398e3148d6038354f8d623b9005ede3a3f83d0c685a8bd9b669a03d52639a",
         "platform": {
            "architecture": "amd64",
            "os": "windows",
            "os.version": "10.0.19041.867"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 1158,
         "digest": "sha256:b88a926ee1a2938c4b00fd608c2c50c8940a773ec6da2efae607094cc1cfe630",
         "platform": {
            "architecture": "amd64",
            "os": "windows",
            "os.version": "10.0.19042.867"
         }
      }
   ]
}
```

`curl -H "Accept: application/vnd.docker.distribution.manifest.list.v2+json" https://mcr.microsoft.com/v2/oss/kubernetes/pause/manifests/3.5`

```json
{
   "schemaVersion": 2,
   "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
   "manifests": [
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 526,
         "digest": "sha256:325bca8ea6bc9cd87b851c8ea0bc07153d24e85e8d1f3201165214142f884345",
         "platform": {
            "architecture": "amd64",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 1158,
         "digest": "sha256:f693069e74d87e478a072ae5869a88964c8025d880774c107bd89184c1553ba3",
         "platform": {
            "architecture": "amd64",
            "os": "windows",
            "os.version": "10.0.17763.1817"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 1158,
         "digest": "sha256:457de298477c17b6c2d05089db5cb274d08f7163d21900a3c85ce0f0818eacb2",
         "platform": {
            "architecture": "amd64",
            "os": "windows",
            "os.version": "10.0.18362.1256"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 1158,
         "digest": "sha256:d7c3fd7bc54a503f4a31531265e5f34ae1aecdd6e71ac2e3d3fc9bb464e1a325",
         "platform": {
            "architecture": "amd64",
            "os": "windows",
            "os.version": "10.0.18363.1440"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 1158,
         "digest": "sha256:92a6264562c5ecabb69c83a64a83f5a57eb0ea3f92a06cb6aafffba13e7b2919",
         "platform": {
            "architecture": "amd64",
            "os": "windows",
            "os.version": "10.0.19041.867"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 1158,
         "digest": "sha256:ce93850bbe689969c018fb6b55bf79f131a08a60712e4d07bfbb59212aabe39c",
         "platform": {
            "architecture": "amd64",
            "os": "windows",
            "os.version": "10.0.19042.867"
         }
      }
   ]
}
```

</details>

**Comment by neolit123** [comment]
@kubernetes/sig-windows-feature-requests

**Comment by jsturtevant** [comment]
> The relationship between the two isn't clear. Are they the same source? Or just the same version by lucky happenstance? Why would I chose one over the other?

The image is built from the same source with fewer targets. For Windows images the files are signed by Microsoft which is requirement for many organizations.  

There use to be divergence between the images because the official pause image didn't have support for Windows but since https://github.com/kubernetes/kubernetes/pull/91452 we can build using the same build tooling.  Since 3.4.1 both gcr and mcr pause images where built for linux and windows, with windows images having signed binaries.

Makes sense we should be clearer about the pause images and what is included in both. It also looks like there are references of 3.5 and 3.4.1 and only be the latest version should be referenced.

**Comment by marosset** [comment]
One difference (that may be big or small depending on the environment you are running in) is that all binaries in the MCR image are Authenticode signed by MS even though they are built from the same source.

**Comment by marosset** [comment]
/close 
This has been addressed with #29063
